.rodare.json

```diff
@@ -5,6 +5,10 @@
             "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
             "orcid": "0000-0001-8704-1769"
         },
+        {
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+            "name": "Poeschel, Franz"
```

Comment:
@franzpoeschel if you have or want to create an [orcid](https://orcid.org/) for your scientific career, now could be a good time :)

Reply:
Cool, added! :)
You can also add it to your HZDR profile here: https://www.hzdr.de/db/UserDB.UserWWW.Run

Reply:
Do you want to be spelled "oe" or ö? :)

Reply:
@franzpoeschel do you want to be spelled "oe" or "ö"? :)